plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -27,28 +27,41 @@ SecRule TX:nextcloud-rule-exclusions-plugin_enabled "@eq 0" "id:9508099,phase:1,
 # you put something like this in crs-setup.conf:
 #
 # SecRule REQUEST_FILENAME "@rx /(?:remote\.php|index\.php)/" \
-#   "id:9508600,\
-#   phase:2,\
-#   t:none,\
-#   nolog,\
-#   pass,\
-#   setvar:'tx.restricted_extensions=.bak/ .config/ .conf/'"
+#     "id:9508600,\
+#     phase:2,\
+#     t:none,\
+#     nolog,\
+#     pass,\
+#     setvar:'tx.restricted_extensions=.bak/ .config/ .conf/'"
 #
 # Large uploads can be modified with SecRequestBodyLimit. Or they
 # can be more controlled by using the following:
 #
 # SecRule REQUEST_URI "@endsWith /index.php/apps/files/ajax/upload.php" \
-#    "id:9508610,\
-#    phase:1,\
-#    t:none,\
-#    nolog,\
-#    ctl:requestBodyLimit=1073741824"
+#     "id:9508610,\
+#     phase:1,\
+#     t:none,\
+#     nolog,\
+#     ctl:requestBodyLimit=1073741824"
+#
+# ctl:requestBodyLimit is not supported in libmodsecurity3, Nginx users can increase max upload size
```

Comment:
Example for Apache is missing also here.

Reply:
As above, httpd is already covered by `requestBodyLimit`.